internal/vulnerability/runner.go

```diff
@@ -48,17 +50,18 @@ func NewVulnerabilityRunner(log *logp.Logger) (VulnerabilityRunner, error) {
 			Quiet:   false,
 			Debug:   true,
 		},
-		VulnerabilityOptions: flag.VulnerabilityOptions{
-			VulnType: []string{trivy_types.VulnTypeOS, trivy_types.VulnTypeLibrary},
+		PackageOptions: flag.PackageOptions{
+			PkgTypes:         []string{trivy_types.PkgTypeOS, trivy_types.PkgTypeLibrary},
+			PkgRelationships: ftypes.Relationships,
```

Comment:
Package relationship has been introduced on https://github.com/aquasecurity/trivy/pull/7237 and using CLI there are default relationship values (all). Using the library directly there is no default value and having 0 relationships and retrieving 0 finginds. Therefore on this commit we added the default value with all relationships